src/ArcResourceBridge/README.md

```diff
@@ -0,0 +1,123 @@
+
+## Module Requirements
+- [Az.Accounts module](https://www.powershellgallery.com/packages/Az.Accounts/), version 2.7.5 or greater
+
+## Authentication
+AutoRest does not generate authentication code for the module. Authentication is handled via Az.Accounts by altering the HTTP payload before it is sent.
+
+## Development
+For information on how to develop for `Az.ArcResourceBridge`, see [how-to.md](how-to.md).
+<!-- endregion -->
+
+### AutoRest Configuration
+> see https://aka.ms/autorest
+
+``` yaml
+branch: 7d6c14d986a67dca3451d7d92d8f6b9416d61fbf
+require:
+  - $(this-folder)/../readme.azure.noprofile.md
+input-file:
+  - $(repo)/specification/resourceconnector/resource-manager/Microsoft.ResourceConnector/stable/2022-10-27/appliances.json
+
+title: ArcResourceBridge
+module-version: 0.1.0
+subject-prefix: $(service-name)
+
+identity-correction-for-post: true
+resourcegroup-append: true
+nested-object-to-string: true
+
+directive:
+  - from: swagger-document 
+    where: $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ResourceConnector/appliances/{resourceName}"].delete.responses
+    transform: >-
+      return {
+        "200": {
+          "description": "Succeeded. The resource bridge was deleted."
+        },
+        "202": {
+          "description": "Accepted. The response indicates the delete operation is performed in the background."
+        },
+        "204": {
+          "description": "NoContent. The response indicates the appliance resource is already deleted."
+        },
+        "default": {
+          "description": "Error response describing why the operation failed.",
+          "schema": {
+            "$ref": "https://github.com/Azure/azure-rest-api-specs/blob/7d6c14d986a67dca3451d7d92d8f6b9416d61fbf/specification/common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+          }
+        }
+      }
+
+  - where:
+      variant: ^Create$|^CreateViaIdentity$|^CreateViaIdentityExpanded$|^Update$|^UpdateViaIdentity$
+    remove: true
+  - where:
+      verb: Set
+    remove: true
+  - where:
+      parameter-name: ResourceName
+    set:
+      parameter-name: Name
+  - where:
+      subject-prefix: ArcResourceBridge
+      subject: Appliance
+    set:
+      subject-prefix: ArcResource
+      subject: Bridge
+  - where:
+      subject-prefix: ArcResourceBridge
+      subject: ApplianceClusterUserCredential
+    set:
+      subject: UserCredential
+  - where:
+      subject-prefix: ArcResourceBridge
+      subject: ApplianceKey
+    set:
+      subject: Credential
+  - where:
+      subject-prefix: ArcResourceBridge
+      subject: ApplianceTelemetryConfig
+    set:
+      subject: TelemetryConfig
+  - where:
+      subject-prefix: ArcResourceBridge
+      subject: ApplianceUpgradeGraph
+    set:
+      subject: UpgradeGraph
+  - where:
+      subject-prefix: ArcResourceBridge
```

Comment:
why not combine directive at line 85 with this one? in the discussion you provided ClusterUserCredential is expected, why you need these two directives?

Reply:
This code has been merged, and an omission occurred when the command name was changed several times.

Reply:
Mentioned in the email before I hide the command "Get - AzArcResourceBridgeApplianceCredential" and"
The Get - AzArcResourceBridgeCredential "but service team don't agree to my idea.